pipeline-manager/pipemgr

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+"""Pipe Manager
+
+Usage:
+    pipemgr create [-f FILE | --file=FILE] [-o OUTFILE | --output=OUTFILE]
+    pipemgr extract [-f FILE | --file=FILE] [-o OUTFILE | --output=OUTFILE] [-c | --clean]
+    pipemgr add-book BOOK [-f FILE | --file=FILE] [--server=SERVER] [-u | --update]
+    pipemgr remove-book BOOK [-f FILE | --file=FILE] [--server=SERVER] [-u | --update] [-y | --yes]
+    pipemgr list-books [-f FILE | --file=FILE]
+    pipemgr (-h | --help)
+
+Options:
+    -h  --help
+        Show help
+
+    -c  --clean
+        Delete stored osbooks before extracting resources from the pipeline
+
+    -u  --update
+        Update the osbooks.yml file after adding/removing booka
+
+    -y  --yes
+        Do not prompt before completing the action
+
+    -f FILE  --file=FILE
+        Use FILE as input for the operation
+
+    -o OUTFILE  --output=OUTFILE
+        Output results to OUTFILE
+
+    --server=SERVER
+        The archive server for the book  [default: cnx.org]
+
+"""
+
+from python_modules import create_pipeline, extract_resources, manage_books
```

Comment:
I saw this just went for ready for review but I might be jumping the gun before you decided to rebase. Don't forget to rename python_modules to something better :wink:.